classes/class-wc-connect-help-provider.php

```diff
@@ -237,8 +244,79 @@ protected function get_services_items() {
 			return array();
 		}
 
+		/**
+		 * Gets the last 10 lines from the WooCommerce Connect log, if it exists
+		 */
+		protected function get_debug_log_data() {
+			$data = new stdClass;
+			$data->key = '';
+			$data->file = '';
+			$data->tail = array();
+
+			if ( method_exists( 'WC_Admin_Status', 'scan_log_files' ) ) {
+				$logs = WC_Admin_Status::scan_log_files();
+
+				foreach ( $logs as $log_key => $log_file ) {
+					if ( "wc-connect-" === substr( $log_key, 0, 11 ) ) {
+						$complete_log = file( WC_LOG_DIR . $log_file );
+						$data->key = $log_key;
+						$data->file = $log_file;
+						$data->tail = array_slice( $complete_log, -10 );
+					}
+				}
+			}
+
+			return $data;
+		}
+
 		protected function get_debug_items() {
-			return array();
+			$debug_items = array();
+
+			// add debug on/off boolean
+			$debug_items[] = (object) array(
+				'key' => 'wcc_debug_on',
+				'title' => 'Debug Logging',
+				'type' => 'boolean',
+				'text' => __( 'Enabled', 'woocommerce' ),
+				'description' => '',
+				'value' => $this->logger->is_logging_enabled(),
+				'saveontoggle' => true
+			);
+
+			// add connect log tail
+			$log_data = $this->get_debug_log_data();
+			if ( count( $log_data->tail ) < 1 ) {
+				$description = '';
+				$log_tail = __( 'Log is empty', 'woocommerce' );
+			} else {
+				$url = add_query_arg(
+					array(
+						'page' => 'wc-status',
+						'tab' => 'logs',
+						'log_file' => $log_data->key
+					),
+					admin_url( 'admin.php' )
+				);
+				$description = sprintf(
+					wp_kses(
+						__( 'Last %d entries <a href="%s">Show full log</a>', 'woocommerce' ),
+						array(  'a' => array( 'href' => array() ) ) ),
+					count( $log_data->tail ),
```

Comment:
Fixed in https://github.com/Automattic/woocommerce-connect-client/pull/376/commits/77e6c33e877d6931f17e4958e855171bc1cf814b